include/z64save.h

```diff
@@ -73,7 +73,7 @@ typedef struct {
     /* 0x0042 */ u8 doubleDefense;   // "life_ability"
     /* 0x0043 */ u8 unk_43;          // "ocarina_round"
     /* 0x0044 */ u8 unk_44;          // "first_memory"
-    /* 0x0046 */ u16 unk_46;         // "memory_warp_point"
+    /* 0x0046 */ u16 owlsHit;         // "memory_warp_point"
```

Comment:
Is this the name we're sticking with? Or are we going with `owlsHitFlags`?

Reply:
Honestly I'm not sure what the best option is here. Could do Flags or Reg (for Register) or Bitfield. Or just leave it as is (with the possible exception of changing the word "owl" to "warpstone" to avoid confusion with kaepora). I don't personally see "owlsHit" as indicating a count (I would do "nOwlsHit for that), but I know others have said they do, so maybe a clarifying word at the end is for the best.

Reply:
imo, `num` at the start of the name would help alot. And is a common pattern we use at least in oot to represent a count of something.
And more importantly, i think the real issue with this name is "hit". The important part is not that you hit it with your sword, but that it became activated *after* you hit it without your sword.
At the expense of a slightly longer name, but for more clarity, my vote would be `numOwlsActivated`.
You dont "activate" kaepora, so imo theres no room for confusion there

Reply:
er wait, if this is flags and not a straight up count then idk about "num". I missed that part of the comment lol

Reply:
This is a bitfield/flags where each bit represents whether a given owl statue is opened or closed. Since i didn't include "n" or "num" to prefix the variable name, that's precisely why I don't think it (incorrectly) reads as a count. But some folks in my stream/the server were saying it does read as a count and that I should postfix it with "Flags" or something to prevent that.

Reply:
I do concur on "activated" instead of "hit" though. I think that's even better than "Opened" since the order things happen in the code is:

1. Registers AC_HIT
2. ObjWarpstone_ClosedIdle calls ObjWarpstone_BeginOpeningCutscene which calls Obj_WarpstonePlayOpeningCutscene.
3. Owl statue model visually opens after timer >= OBJ_WARPSTONE_TIMER_OPENED.
4. Owl statue is activated and ObjWarpstone_OpenedIdle is called after time >= OBJ_WARPSTONE_TIMER_DONE.

Reply:
Perhaps I should rename that enum from _DONE to _ACTIVATED, now that I'm thinking about it.

Reply:
I changed the name of this field to owlActivationFlags. That was the best averaged name I could come up with taking into account everyone's suggestions. I'll open a new PR for it since this one got merged before I could get to it.